projects/wave-core-new/src/lib/time/time-slider/time-slider.component.ts

```diff
@@ -0,0 +1,282 @@
+import {Component, OnInit, ChangeDetectionStrategy, ViewChild, ChangeDetectorRef, OnDestroy} from '@angular/core';
+import {DataSet} from 'vis-data/standalone';
+import {DateType, Timeline} from 'vis-timeline/standalone';
```

Comment:
Could you use `vis-timeline/esnext`. The docs say `standalone` is bloaty and our bundle size is already quite large.

Here is an example where you have to put the CSS and some other js file into the angular.json:
https://github.com/visjs/vis-timeline/issues/446#issuecomment-618419790

Reply:
2ba39da403827e26f3232b9e3fed2ba81b97a8dd